stable/kong/templates/migrations.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+  annotations:
+    helm.sh/hook: pre-install,pre-upgrade
```

Comment:
This will always fail since the this job requires that the Postgres DB is up and running.